lib/rules/rangeContextNodeParser.js

```diff
@@ -23,6 +26,9 @@ function getRangeContextOperators(node) {
 	return operators.sort((a, b) => b.length - a.length);
 }
 
```

Comment:
```ts
const before = node[match.startIndex - 1];
```

This line has been added via #4581, but I think the type of `node` should be `import('postcss-media-query-parser').Node`. So, the type error occurs like this:

> Element implicitly has an 'any' type because expression of type 'number' can't be used to index type 'Node'.

Why? 🤔

Reply:
I've fixed the type-check error via the c26238b1 commit. Perhaps, I guess that `(before === '>' || before === '<')` in the `if` condition may exclude a pattern like `<<` or `>>` etc.